src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -38,7 +36,7 @@ public Iterable findAll(){
     }
 
     @GetMapping("/author/{bookAuthor}")
-    public Book findByAuthor(@PathVariable String bookAuthor) {
+    public Optional<Book> findByAuthor(@PathVariable String bookAuthor) {
```

Comment:
Remember that you should return the Book directly, or an exception if it's not found :) (If you modified it in another PR you can dismiss my comment 😅 )